### DISCUSSION TITLE
Elasticsearch metric name

### DISCUSSION THREAD

**Discussion by nhadsall** [grafana/grafana#61185] [Dashboards, Panels & Visualizations]
We REALLY should be able to name the various metrics returned.

Sum 1, Sum 2, Sum 3, etc. just don't work. The issue is when there is NO "Sum 2" then Sum 3 becomes Sum 2... and there's no simple way to name.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I believe this is something that can be accomplished with the "Organize Fields by Name" transformation, which lets you rename fields. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if this solution does not work and you would like to request the feature.
